Meadowlark-js/tests/e2e/scenarios/SecurityValidation.test.ts

```diff
@@ -220,6 +221,7 @@ describe('given the existance of two vendors and one host', () => {
 
     describe('when the same vendor updates the resource', () => {
       it('should return success', async () => {
+        resourceBodyPUTUpdated.id = resourceLocation.split('/').pop();
```

Comment:
Would you mind renaming such variables to always be camel cased? In this case `resourceBodyPUTUpdated` -> `resourceBodyPutUpdated`